enarx-keep/tests/integration_tests.rs

```diff
@@ -30,6 +30,17 @@ fn run_test(bin: &str, status: i32) -> std::process::Child {
         .spawn()
         .expect("failed to run test bin");
 
+    if let Some(input) = input {
+        child
+            .stdin
+            .as_mut()
+            .unwrap()
+            .write_all(input.as_ref())
```

Comment:
```suggestion
            .write_all(input.into().as_ref())
```

Reply:
I had to use `to_string()` so that it didn't resolve to `T`.